README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My complete name is *Miguel Ángel Rosales Legorreta*
+[My linkedin profile](www.linkedin.com)
```

Comment:
I'm assuming these two lines were meant to be separate, rather than one long string (as in the rendered version). In order to start a new line or paragraph, make sure to **end the previous line with two spaces**